lib/refactorings/providers/open-ai/OpenAIClient.js

```diff
@@ -36,14 +47,28 @@ export default class OpenAIClient {
         }
       ],
       model: this._model,
+      seed: this._seed,
+      temperature: this._temperature,
       tool_choice: 'auto',
       tools
     });
 
+    // console.log(JSON.stringify(response, null, 2));
+
```

Comment:
```suggestion
```

Reply:
Resolved in [a26efd7](https://github.com/bpmn-io/refactorings/pull/6/commits/a26efd77c35a2daf8dd852014da946c296132f47)